tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -4252,6 +4253,10 @@ TEST (test_tensor_filter, framework_auto_ext_tflite_no_permission_n)
   const gchar *fw_name = NULL;
   GstElement *gstpipe;
 
+  /** If the user is root, skip this test */
+  if (getuid () == 0)
```

Comment:
`geteuid`?

Reply:
It says both `getuid` and `geteuid` return `0` for the root. `geteuid` would be more proper?

Reply:
There's a scenario `euid` can be zero even if `uid` is not zero, and it also has permissions to behave like a root.

Reply:
Thanks! You're right. I updated with `geteuid`